doc/observable.md

```diff
@@ -382,7 +382,7 @@ var subscription = observable.subscribe(x => console.log(x));
 subscription.unsubscribe();
 ```
 
-<span class="informal">When you subscribe, you get back a Subscription, which represents the ongoing execution. Just call `unsubscribe()` to cancel the execution.</span>
+<span class="informal">When you subscribe, you get back a Subscription, which represents the ongoing execution. Calling `unsubscribe()` will cancel the execution.</span>
```

Comment:
I agree this wasn't one of those places where I found it condescending. I found it to be an answer to the question "Hey how do I unsubscribe?" even though I wasn't asking. I feel way less strongly about this than my other changes. Will put it back how it was.